### DISCUSSION TITLE
Where are services hosted?

### DISCUSSION THREAD

**Discussion by deusaquilus** [community/community#26561] [Actions]
<p>Are services hosted in the runner VM or are they in a different one? E.g. If I have a service that takes 3GB of memory, will that memory be taken from the runner VM or a different place?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Anything started via <code>jobs.&lt;job_id&gt;.services</code> is a Docker container on the runner, so yes, memory for services will be taken out of the RAM of the runner.</p>
